cypress/integration/footer-feature.spec.js

```diff
@@ -0,0 +1,236 @@
+                .click();
+
+            cy
+                .get('a[name=privacy-policy]')
+                .should('exist');
+        });
+
+        it('should conatin a text link containing "Accessibility" that points to the policies page', () => {
+            cy
+                .contains('a', 'Accessibility')
+                .should('have.attr', 'href', '/about-us/policies.html#accessibility-statement')
+                .click();
+
+            cy
+                .get('a[name=accessibility-statement]')
+                .should('exist');
+        });
+
+    });
+
+    describe('Adjust Contrast', () => {
+        // Backup the clear function.
+        // This is going to be set to null below...
+        const clear = Cypress.LocalStorage.clear;
+
+        const assertIsHighContrast = () => {
+
+            cy
+                .window()
+                .its('localStorage')
+                .its('isHighContrast')
+                .should('eq', 'true');
+
+        };
+
+        const assertIsNotHighContrast = () => {
+
+            cy
+                .window()
+                .its('localStorage')
+                .its('isHighContrast')
+                .should('eq', 'false');
+
+        };
+
+        before(() => {
+            cy.visit(constants.rootUrl);
+        });
+
+        it('should contain a contraster', () => {
+            cy
+                .get(constants.selectors.contraster)
+                .should('exist');
+        });
+
+        it('should toggle and change the page colors to high contrast', () => {
+            cy
+                .get(constants.selectors.contraster)
+                .click();
+
+            assertIsHighContrast();
+        });
+        
+        it('should toggle and change page colors back', () => {
+            cy
+                .get(constants.selectors.contraster)
+                .click();
+
+            assertIsNotHighContrast();
+        });
+        
+        it('should toggle when "Adjust Contrast" is clicked', () => {
+            cy
+                .get(constants.selectors.contraster)
+                .contains('Adjust Contrast')
+                .click();
+
+            assertIsHighContrast();
+
+            // Cypress automatically clears local storage between tests
+            // Temporarily removing the clear function allows local storage to persist
+            // Customizable lifecycle event will be available in a future release
+            // https://github.com/cypress-io/cypress/issues/686
+            Cypress.LocalStorage.clear = null;
+        });
+        
+        it('should keep contrast colors on other pages', () => {
+            cy
+                .get(`${constants.selectors.calloutDivs}`)
+                .contains('Ask a Librarian')
+                .click();
+
+            assertIsHighContrast();
+
+            // Re-assign the clear function
+            Cypress.LocalStorage.clear = clear;
```

Comment:
Think you can throw this in an afterEach

```javascript
afterEach(() => {
Cypress.LocalStorage.clear = clear;
});
```

would put this after the before call above.